dockers/docker-dhcp-relay/cli-plugin-tests/test_show_dhcpv6_helper.py

```diff
@@ -34,7 +34,8 @@ def test_show_dhcpv6_helper(self, test_name, test_data, fs):
             fs.create_file(DBCONFIG_PATH) 
         MockConfigDb.set_config_db(test_data["config_db"])
         runner = CliRunner()
-        table = MockConfigDb.get_table(self, "DHCP_RELAY")
+        config_db = MockConfigDb()
+        table = config_db.get_table("DHCP_RELAY")
```

Comment:
Is this change related to IP2ME?

Reply:
Yes, the short summary is that it fixes a bug in how MockConfigDb is called from `test_show_dhcpv6_helper`. Without this fix  `MockConfigDb.get_table` will crash when referencing it's `self`. The previous author seems to have assumed that get_table is static, which it is not, and accidentally passed `self` as an argument which causes all kinds of weirdness.

Reply:
Thanks for addressing this issue! just figure that this change should be in a separate PR.

Reply:
Rebased upon new master to remove this part from this PR. Please re-review @yxieca :-).